scripts/plotting/global_latlon_map.py

```diff
@@ -251,11 +252,15 @@ def global_latlon_map(adfobj):
 
                     # difference: each entry should be (lat, lon)
                     dseasons[s] = mseasons[s] - oseasons[s]
+                    
+                    # percent change
+                    #pseasons[s] = (abs(mseasons[s] - oseasons[s])/((mseasons[s] + oseasons[s])/2.0))*100.0
```

Comment:
Could you remove this commented out line?

Reply:
Done